Stripe/StripeiOSTests/PaymentSheet+APITest.swift

```diff
@@ -209,34 +209,149 @@ class PaymentSheetAPITest: XCTestCase {
     }
 
     func testPaymentSheetLoadWithDeferredIntent() {
```

Comment:
Hmm we have `testPaymentSheetLoadAndConfirmWithPaymentIntent` today, and since this test is also confirming now, we should try to keep the pattern.

```suggestion
    func testPaymentSheetLoadAndConfirmWithDeferredIntent() {
```

Reply:
oop yeah, meant to rename this.